test/index.js

```diff
@@ -18,6 +18,8 @@ global.d      = new Date();
 Promise.longStackTraces();
 
 describe('Query Building Tests', function() {
+  this.timeout(process.env.KNEX_TEST_TIMEOUT || 5000);
```

Comment:
According to [Mocha Docs](http://mochajs.org/#timeouts) calling `.timeout(ms)` in a describe will apply that timeout for all nested tests in that suite. So maybe it'll work by adding `.timeout()` under `describe('Query Building Tests')` / `describe('Integration Tests')` ?

I know it's already merged, just figured I'd mention. :smile: